docker-compose-inz-and-demo.yml

```diff
@@ -0,0 +1,125 @@
+---
```

Comment:
I could use this docker compose file to successfully get up a irma setup using inz-demo + irma-demo schemes. Then I could issue inz-demo credentials and irma-demo credentials.